code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -3820,4 +3820,28 @@ obj/item/chems/food/snacks/dango
 /obj/item/chems/food/snacks/old/taco
 	name = "taco"
 	desc = "Interestingly, the shell has gone soft and the contents have gone stale."
-	icon_state = "ancient_taco"
\ No newline at end of file
+	icon_state = "ancient_taco"
+
+/obj/item/chems/food/snacks/pelmen
+	name = "meat pelmen"
+	desc = "Raw meat appetizer."
+	icon_state = "pelmen"
+	filling_color = "#ffffff"
+	center_of_mass = @"{'x':16,'y':16}"
+	bitesize = 2
+
+/obj/item/chems/food/snacks/pelmen/Initialize()
+	. = ..()
+	reagents.add_reagent(/decl/material/liquid/nutriment/protein, 1)
```

Comment:
There's also a bread reagent (only really relevant on scav where it makes yinglets ill): `/decl/material/liquid/nutriment/bread`